.ci/Jenkinsfile

```diff
@@ -36,6 +35,7 @@ pipeline {
         deleteDir()
         gitCheckout(basedir: "${BASE_DIR}")
         stash allowEmpty: true, name: 'source', useDefaultExcludes: false
+        setEnvVar('ELASTIC_LAYER_NAME', "elastic-apm-extension${getVersion()}")
```

Comment:
I think we need to include a suffix for the architecture, otherwise the ARNs for x86_64 and arm64 are indistinguishable.